contracts/axelar-gateway/tests/test.rs

```diff
@@ -490,3 +492,346 @@ fn upgrade_unauthorized() {
     assert_invoke_auth_err!(not_owner, client.try_upgrade(&new_wasm_hash));
     assert_invoke_auth_err!(client.operator(), client.try_upgrade(&new_wasm_hash));
 }
+
+#[test]
+#[should_panic]
```

Comment:
add expected panic message

Reply:
done. the test actually would have returned info on the error but i was passing in a i32 where a u64 was needed (defaulted to that) so that's why it was failing.